### PR TITLE
Updated German translations of two new strings (second try)

### PR DESCRIPTION
**Pull request by minusLik**
* Hide Inline ads as a benefit of being a supporter
* Improved wording for K&P perving benefit

### CODE REVIEW

**Comment by minusLik** [comment]
For the records: This is a replacement for #280 which was a mess because I had a problem rebasing my fork repo before I made that pull request. I cleaned up and it looks better now.

File: locales/de.yml [hunk 1]
```diff
@@ -482,14 +482,14 @@ de:
   support:
     benefits:
       item_1: Eine sexy <span>I Support FetLife</span>-Plakette auf Deinem Profil.
-      item_2: Perv <span>over 5,000 of today’s most popular</span> pics, vids, and writings.
+      item_2: Durchstöbere täglich <span>über 5.000 der beliebtesten</span> Bilder, Videos und Texte.
```
Comment (nielsenramon) [inline locales/de.yml:485]: Is there a better word for `Texte`? `Artikel` perhaps?

Comment (minusLik) [reply locales/de.yml:485]: Maybe, but it's not »Artikel«. Artikel usually refers to journalistic work (just like its English counterpart), which erotica and journal entries clearly aren't. I can offer »Schriften«, although this sounds quite uptown for my taste. However, I'll put it in and wait for complaints 😉

Sorry for the late response BTW, I've been occupied.

Comment (nielsenramon) [reply locales/de.yml:485]: @minusLik All good! mind fixing the conflicts as well so we can merge please? 🙏

Comment (minusLik) [reply locales/de.yml:485]: That's done. I'll create a new merge request so that the new translations can come along, too.

**Pull request closed without merging**